locales/cmd/login/active.en.toml

```diff
@@ -12,6 +12,9 @@ one = '''
 Log in securely to RHOAS through a web browser.
 
 This command opens your web browser, where you can enter your credentials.
+
+Alternatively you can log in using an offline-token by passing the "--token" flag, which can be obtained at {{.OfflineTokenURL}}.
+Note: token-based login is not supported by the Kafka "topic" and "consumergroup" subcommands.
```

Comment:
```suggestion
Note: token-based login is not supported by the Kafka "topic" and "consumergroup" subcommands and it is not recommended for personal use.
```

Reply:
I'm unsure about highlighting that this is not recommended for personal use.

Reply:
Sure. That is opiniated take of mine. Please ignore